### PR TITLE
README update for repository

### PR DESCRIPTION
**Pull request by nik-dange**
- Updated README to reflect the repository better. Let me know if anything should be added/removed

### CODE REVIEW

**Comment by nik-dange** [comment]
Also added some miscellaneous housekeeping edits to close some issues

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
-# Nextra Docs
+# hack-website
```
Comment (nishantbalaji) [inline README.md:1]: capitalize, name it `ACM Hack Website`

File: README.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 
-This is a template for creating documentation with [Nextra](https://nextra.site).
+Welcome to ACM Hack's main website! This website serves as an introduction to the Hack community, and provides information about our upcoming events and resources. It's also the primary host of our Hack School curriculum, which is a series of workshops designed to teach students the fundamentals of full-stack development and software engineering.
+
+The content on this website is accessible to anyone, regardless of acceptance to Hack School. Feel free to use this for ACM Projects, personal projects, or just to learn something new!
+
+This project was bootstrapped with [Nextra](https://nextra.site).
 
 
 [**Live Demo (https://hack-website-acmucsd.vercel.app/)**](https://hack-website-acmucsd.vercel.app/)
```
Comment (nishantbalaji) [inline README.md:10]:
```suggestion
Check it out @ [hack.acmucsd.com](https://hack.acmucsd.com/)
```

Comment (nishantbalaji) [reply README.md:10]: Also move this to the top right under the title

File: pages/hack-school/logistics.mdx
Comment (nishantbalaji) [inline pages/hack-school/logistics.mdx:1]: normally i would say this should be in its own PR but its fine since its so small

File: README.md [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
 
-## Local Development
+## Contributing
```
Comment (nishantbalaji) [inline README.md:12]: mention they can also make a pr if they want? idk

File: README.md [hunk 4]
```diff
@@ -1,13 +1,20 @@
-# Nextra Docs
+# hack-website
 
-This is a template for creating documentation with [Nextra](https://nextra.site).
+Welcome to ACM Hack's main website! This website serves as an introduction to the Hack community, and provides information about our upcoming events and resources. It's also the primary host of our Hack School curriculum, which is a series of workshops designed to teach students the fundamentals of full-stack development and software engineering.
+
+The content on this website is accessible to anyone, regardless of acceptance to Hack School. Feel free to use this for ACM Projects, personal projects, or just to learn something new!
+
+This project was bootstrapped with [Nextra](https://nextra.site).
```
Comment (nishantbalaji) [inline README.md:7]: and deployed on Vercel

**Review by nishantbalaji — CHANGES_REQUESTED** [review]
looks good overall but i have nits 🍡

File: pages/hack-school/logistics.mdx
Comment (nik-dange) [reply pages/hack-school/logistics.mdx:1]: thanks :3

File: README.md [hunk 5]
```diff
@@ -1,13 +1,19 @@
...
+The content on this website is accessible to anyone, regardless of acceptance to Hack School. Feel free to use this for ACM Projects, personal projects, or just to learn something new!
 
-## Local Development
+This project was bootstrapped with [Nextra](https://nextra.site) and deployed with [Vercel](https://vercel.com).
 
-Prerequisites
+## Contributing
+
+We welcome any contributions from the ACM community! If you have ideas for new content to add, feel free to contact us in the [ACM Discord](https://acmurl.com/discord) or [ACM Hack Discord](https://acmurl.com/hack-discord) by pinging the `@Hack` role, or directly opening up a pull request. If you notice any content errors or have suggestions
+for improvements, feel free to raise an issue on our repository. Make sure to follow the instructions below to set up your local environment for development.
```
Comment (nishantbalaji) [inline README.md:14]: same line for these

**Review by nishantbalaji — APPROVED** [review]
🚢 🌊

File: README.md (hunk 5 above)
Comment (nik-dange) [reply README.md:14]: what does that mean

**Pull request merged**